pkg/operator/controllers/guardrails/policies/gktemplates-src/aro-deny-clusterversion-upgradeconfig/src_test.rego

```diff
@@ -0,0 +1,206 @@
+package arodenyclusterversionupgradeconfig
+
```

Comment:
I see you have combine `clusterversion` and `upgradeconfig` logics together as one constraint template, so do we need some more test cases here to cover `clusterversion` checks?